build.gradle.kts

```diff
@@ -0,0 +1,101 @@
+group = "no.nav.syfo"
+version = "0.0.1"
+
+val flywayVersion = "10.17.2"
+val hikariVersion = "5.1.0"
+val postgresVersion = "42.7.4"
+val postgresEmbeddedVersion = "2.0.7"
+val kafkaVersion = "3.7.0"
+val logbackVersion = "1.5.7"
+val logstashEncoderVersion = "7.4"
+val micrometerRegistryVersion = "1.12.2"
+val jacksonDatatypeVersion = "2.17.2"
+val ktorVersion = "3.0.1"
+val spekVersion = "2.0.19"
+val mockkVersion = "1.13.12"
+val nimbusJoseJwtVersion = "9.40"
+val kluentVersion = "1.73"
+
+plugins {
+    kotlin("jvm") version "2.0.20"
+    id("com.gradleup.shadow") version "8.3.1"
+    id("org.jlleitschuh.gradle.ktlint") version "11.6.1"
+}
+
+repositories {
+    mavenCentral()
+    maven(url = "https://packages.confluent.io/maven/")
+}
+
+dependencies {
+    implementation(kotlin("stdlib"))
+    implementation(kotlin("reflect"))
+
+    implementation("io.ktor:ktor-client-apache:$ktorVersion")
+    implementation("io.ktor:ktor-client-content-negotiation:$ktorVersion")
+    implementation("io.ktor:ktor-serialization-jackson:$ktorVersion")
+    implementation("io.ktor:ktor-server-auth-jwt:$ktorVersion")
+    implementation("io.ktor:ktor-server-call-id:$ktorVersion")
+    implementation("io.ktor:ktor-server-content-negotiation:$ktorVersion")
+    implementation("io.ktor:ktor-server-netty:$ktorVersion")
+    implementation("io.ktor:ktor-server-status-pages:$ktorVersion")
+
+    // Logging
+    implementation("ch.qos.logback:logback-classic:$logbackVersion")
+    implementation("net.logstash.logback:logstash-logback-encoder:$logstashEncoderVersion")
+
+    // Metrics and Prometheus
+    implementation("io.ktor:ktor-server-metrics-micrometer:$ktorVersion")
+    implementation("io.micrometer:micrometer-registry-prometheus:$micrometerRegistryVersion")
+
+    // Database
+    implementation("org.postgresql:postgresql:$postgresVersion")
+    implementation("com.zaxxer:HikariCP:$hikariVersion")
+    implementation("org.flywaydb:flyway-database-postgresql:$flywayVersion")
+    testImplementation("io.zonky.test:embedded-postgres:$postgresEmbeddedVersion")
+
+    // Kafka
+    implementation("org.apache.kafka:kafka_2.13:$kafkaVersion")
+
+    // (De-)serialization
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jacksonDatatypeVersion")
+
+    // Tests
+    testImplementation("io.ktor:ktor-server-test-host:$ktorVersion")
+    testImplementation("io.mockk:mockk:$mockkVersion")
+    testImplementation("io.ktor:ktor-client-mock:$ktorVersion")
+    testImplementation("com.nimbusds:nimbus-jose-jwt:$nimbusJoseJwtVersion")
+    testImplementation("org.amshove.kluent:kluent:$kluentVersion")
```

Comment:
Vi kan evt ta en diskusjon på om vi skal droppe kluent og spek i denne appen (siden de bibliotekene er døde) og heller gå for JUnit i testene.

Reply:
Kan godt merge så får vi ta en runde på det før vi lager mer kode og tester her